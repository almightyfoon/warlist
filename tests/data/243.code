cb501b_-0v45fA4e4efI4h4ncR1j1ji3___-0vcr4c4efI2Q2Q4h4h4n1i1j1j1j2N