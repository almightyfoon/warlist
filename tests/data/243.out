https://conflictchamber.com/#cb501b_-0v45jQjMjMfIk64njOkckci3___-0vcr4cjMfI2Q2Qk6k64nkbkckckc2N

Minion Army - 72 / 75 points
[Theme] The Blindwater Congregation

[Calaban 1] Calaban, the Grave Walker [+29]
 - Blind Walker [10]
 - Ironback Spitter [14]
 - Ironback Spitter [14]
Bog Trog Mist Speaker [4]
Thrullg [0(5)]
Totem Hunter [0(6)]
Gatorman Bokor & Bog Trog Swamp Shamblers [10]
Gatorman Posse (max) [16]
Gatorman Posse (max) [16]
Sacral Vault [17]

Minion Army - 71 / 75 points
[Theme] The Blindwater Congregation

[Rask 1] Rask [+27]
 - Blackhide Wrastler [16]
 - Ironback Spitter [14]
Bog Trog Mist Speaker [4]
Feralgeist [2]
Feralgeist [2]
Thrullg [0(5)]
Thrullg [0(5)]
Totem Hunter [0(6)]
Gatorman Posse (min) [10]
Gatorman Posse (max) [16]
Gatorman Posse (max) [16]
Gatorman Posse (max) [16]
Swamp Gobber Bellows Crew [2]
